admin_interface/credentials.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+/*Database credentials*/
+
+define("DBHOST", "localhost");
+define("DBNAME", "humanface");
+define("DBUSER", "humanface");
+define("DBPASS", "humanface");
```

Comment:
You shouldn't put credential information in the Github -- put it in .gitignore file so it is not included in the pull request